core/src/main/java/com/novoda/noplayer/player/PlayerFactory.java

```diff
@@ -98,4 +96,27 @@ static UnableToCreatePlayerException unknownPlayerType(PlayerType playerType) {
 
     }
 
+    static class ExoPlayerCreator {
+
+        Player createExoPlayer(Context context, DrmSessionCreator drmSessionCreator) {
+            RendererFactory rendererFactory = createRendererFactory(context, drmSessionCreator);
+            ExoPlayerFacade exoPlayerFacade = new ExoPlayerFacade(rendererFactory);
+            return new ExoPlayerImpl(exoPlayerFacade);
+        }
+
+        private RendererFactory createRendererFactory(Context context, DrmSessionCreator drmSessionCreator) {
+            return new RendererFactory(context, drmSessionCreator);
+        }
+
+    }
+
+    static class MediaPlayerCreator {
+
+        Player createMediaPlayer(Context context) {
+            AndroidMediaPlayerFacade androidMediaPlayer = new AndroidMediaPlayerFacade(context);
+            return new AndroidMediaPlayerImpl(androidMediaPlayer);
+        }
+
+    }
+
```

Comment:
These could use merging in from `master` and doing a `cmd-alt-l` with the new project settings :)